### PR TITLE
config: increase grace_period for docs SIG repos

### PR DESCRIPTION
**Pull request by hi-rustin**
Increase grace_period for docs SIG repos.

### CODE REVIEW

File: prow/config/plugins.yaml [hunk 1]
```diff
@@ -58,11 +58,13 @@ require_matching_label:
...
+    grace_period: 10s
   - missing_label: missing-translation-status
     org: pingcap
     repo: docs-cn
     prs: true
     regexp: (^translation/)|(type/(.*)-cherry-pick$)
+    grace_period: 10s
   - missing_label: missing-translation-status
     org: pingcap
     repo: docs-dm
```
Comment (ran-huang) [inline prow/config/plugins.yaml:70]: Please configure grace_period for docs-dm as well.

Comment (hi-rustin) [reply prow/config/plugins.yaml:70]: Fixed.

**Pull request merged**